src/design-system/Metric.tsx

```diff
@@ -0,0 +1,51 @@
+import numeral from "numeral";
+import React from "react";
+import styled from "styled-components";
+
+import Colors from "../design-system/Colors";
+
+const MetricContainer = styled.div`
+  border-radius: 4px;
+  display: flex;
+  flex-direction: column;
+  margin: 5px;
+  background-color: ${Colors.lightGray};
+  flex: 1 1 30%;
+  height: 96px;
+  font-family: "Poppins", sans serif;
+  text-align: center;
+  line-height: 24px;
+  justify-content: center;
+`;
+
+const Value = styled.div`
+  font-size: 24px;
+  font-weight: 600;
+  color: ${Colors.burgundy};
+`;
+
+const Label = styled.div`
+  font-size: 12px;
+  font-weight: normal;
+  color: ${Colors.opacityForest};
+`;
+
+interface Props {
+  value: number | string | React.ReactElement;
+  label?: string;
+}
+
+const Metric: React.FC<Props> = (props) => {
+  const { value, label } = props;
+
+  return (
+    <MetricContainer>
+      <Value>
+        {typeof value === "number" ? numeral(value).format("0,0") : value}
```

Comment:
Curious: does this `typeof` check work for the function call from `facilitiesWithCases` below? It evaluates the return type and not the function type itself?

Reply:
The function call returns a string so `typeof value` check works for that metric as well.

<img width="747" alt="Screen Shot 2020-05-15 at 3 06 27 PM" src="https://user-images.githubusercontent.com/5872651/82100063-b2255d80-96bd-11ea-8fac-992e4934e702.png">